### PR TITLE
[Add-ins][All hosts] Add links to the Dev Program

### PR DESCRIPTION
**Pull request by mscharlock**
Putting some cross-links to the dev program seems to me like it would be a good idea since we rarely expose it in our docs. :)

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[106a51e](https://github.com/OfficeDev/office-js-docs-pr/commits/106a51e6aa51104738a5ddf24066c58de4c2a08b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/add-in-development-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/concepts/add-in-development-best-practices.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/concepts/add-in-development-best-practices?branch=pr-en-us-2265) |
[docs/develop/develop-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/develop/develop-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/develop/develop-overview?branch=pr-en-us-2265) |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-overview?branch=pr-en-us-2265) |
[docs/excel/excel-add-ins-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/excel/excel-add-ins-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/excel-add-ins-overview?branch=pr-en-us-2265) |
[docs/onenote/onenote-add-ins-programming-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/onenote/onenote-add-ins-programming-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/onenote/onenote-add-ins-programming-overview?branch=pr-en-us-2265) |
[docs/outlook/outlook-add-ins-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/outlook/outlook-add-ins-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/outlook-add-ins-overview?branch=pr-en-us-2265) |
[docs/overview/explore-with-script-lab.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/overview/explore-with-script-lab.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/overview/explore-with-script-lab?branch=pr-en-us-2265) |
[docs/overview/learning-path-beginner.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/overview/learning-path-beginner.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/overview/learning-path-beginner?branch=pr-en-us-2265) |
[docs/overview/learning-path-transition.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/overview/learning-path-transition.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/overview/learning-path-transition?branch=pr-en-us-2265) |
[docs/overview/office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/overview/office-add-ins.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/overview/office-add-ins?branch=pr-en-us-2265) |
[docs/overview/patterns-and-practices-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/overview/patterns-and-practices-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/overview/patterns-and-practices-overview?branch=pr-en-us-2265) |
[docs/overview/set-up-your-dev-environment.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/overview/set-up-your-dev-environment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/overview/set-up-your-dev-environment?branch=pr-en-us-2265) |
[docs/powerpoint/powerpoint-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/powerpoint/powerpoint-add-ins.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/powerpoint/powerpoint-add-ins?branch=pr-en-us-2265) |
[docs/project/project-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/project/project-add-ins.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/project/project-add-ins?branch=pr-en-us-2265) |
[docs/resources/resources-links-help.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/resources/resources-links-help.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/resources/resources-links-help?branch=pr-en-us-2265) |
[docs/word/word-add-ins-core-concepts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/word/word-add-ins-core-concepts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/word/word-add-ins-core-concepts?branch=pr-en-us-2265) |
[docs/word/word-add-ins-programming-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/word/word-add-ins-programming-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/word/word-add-ins-programming-overview?branch=pr-en-us-2265) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C10%5C14%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202010142235421952-2265%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=FtiVhTxdgKZdmgb3HVQgSmGjPdNUCtcndlti7T1wL3M%3D&st=2020-10-14T22%3A31%3A17Z&se=2020-11-14T22%3A36%3A17Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/outlook/outlook-add-ins-overview.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ To get started building Outlook add-ins, try the following.
 
 
 ## See also
-
+-[Learn about the Microsoft 365 Developer Program](https://developer.microsoft.com/microsoft-365/dev-program)
```
Comment (alison-mk) [inline docs/outlook/outlook-add-ins-overview.md:78]: Nit: adding a space after the dash to be consistent with the others in the list 
```suggestion
- [Learn about the Microsoft 365 Developer Program](https://developer.microsoft.com/microsoft-365/dev-program)
```

File: docs/overview/explore-with-script-lab.md [hunk 2]
```diff
@@ -81,3 +81,4 @@ When you're ready to create your first Office Add-in, try out the quick start fo
 - [Learn more about Script Lab](https://github.com/OfficeDev/script-lab#script-lab-a-microsoft-garage-project)
 - [Join the Microsoft 365 developer program](https://developer.microsoft.com/office/dev-program)
 - [Developing Office Add-ins](../develop/develop-overview.md)
+- [Learn about the Microsoft 365 Developer Program](https://developer.microsoft.com/microsoft-365/dev-program)
```
Comment (alison-mk) [inline docs/overview/explore-with-script-lab.md:84]: Line 84 seems similar to line 82, do we need both links?

**Review by alison-mk — APPROVED** [review]
Makes sense to me! Looks good. Two minor suggestions/questions.

**Comment by ElizabethSamuel-MSFT** [comment]
@mscharlock Note that Developer Program is in the header on docs though:
image

**Comment by o365devx** [comment]
Docs Build status updates of commit _[3f79de5](https://github.com/OfficeDev/office-js-docs-pr/commits/3f79de5345041bce66d8c33d6573f4998adf5350)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/add-in-development-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/concepts/add-in-development-best-practices.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/concepts/add-in-development-best-practices?branch=pr-en-us-2265) |
[docs/develop/develop-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/develop/develop-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/develop/develop-overview?branch=pr-en-us-2265) |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-overview?branch=pr-en-us-2265) |
[docs/excel/excel-add-ins-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/excel/excel-add-ins-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/excel-add-ins-overview?branch=pr-en-us-2265) |
[docs/onenote/onenote-add-ins-programming-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/onenote/onenote-add-ins-programming-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/onenote/onenote-add-ins-programming-overview?branch=pr-en-us-2265) |
[docs/outlook/outlook-add-ins-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/outlook/outlook-add-ins-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/outlook-add-ins-overview?branch=pr-en-us-2265) |
[docs/overview/explore-with-script-lab.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/overview/explore-with-script-lab.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/overview/explore-with-script-lab?branch=pr-en-us-2265) |
[docs/overview/learning-path-beginner.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/overview/learning-path-beginner.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/overview/learning-path-beginner?branch=pr-en-us-2265) |
[docs/overview/learning-path-transition.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/overview/learning-path-transition.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/overview/learning-path-transition?branch=pr-en-us-2265) |
[docs/overview/office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/overview/office-add-ins.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/overview/office-add-ins?branch=pr-en-us-2265) |
[docs/overview/patterns-and-practices-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/overview/patterns-and-practices-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/overview/patterns-and-practices-overview?branch=pr-en-us-2265) |
[docs/overview/set-up-your-dev-environment.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/overview/set-up-your-dev-environment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/overview/set-up-your-dev-environment?branch=pr-en-us-2265) |
[docs/powerpoint/powerpoint-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/powerpoint/powerpoint-add-ins.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/powerpoint/powerpoint-add-ins?branch=pr-en-us-2265) |
[docs/project/project-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/project/project-add-ins.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/project/project-add-ins?branch=pr-en-us-2265) |
[docs/resources/resources-links-help.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/resources/resources-links-help.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/resources/resources-links-help?branch=pr-en-us-2265) |
[docs/word/word-add-ins-core-concepts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/word/word-add-ins-core-concepts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/word/word-add-ins-core-concepts?branch=pr-en-us-2265) |
[docs/word/word-add-ins-programming-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/word/word-add-ins-programming-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/word/word-add-ins-programming-overview?branch=pr-en-us-2265) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C10%5C20%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202010201907109508-2265%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=VOJmpEq%2BwGukcUBbJ4WsfCprsp3GzBCPVbTkxgbnMeI%3D&st=2020-10-20T19%3A02%3A53Z&se=2020-11-20T19%3A07%3A53Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[541ba52](https://github.com/OfficeDev/office-js-docs-pr/commits/541ba523a8884a3171e325b9df5a905eb7c01e93)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/add-in-development-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/concepts/add-in-development-best-practices.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/concepts/add-in-development-best-practices?branch=pr-en-us-2265) |
[docs/develop/develop-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/develop/develop-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/develop/develop-overview?branch=pr-en-us-2265) |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-overview?branch=pr-en-us-2265) |
[docs/excel/excel-add-ins-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/excel/excel-add-ins-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/excel-add-ins-overview?branch=pr-en-us-2265) |
[docs/onenote/onenote-add-ins-programming-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/onenote/onenote-add-ins-programming-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/onenote/onenote-add-ins-programming-overview?branch=pr-en-us-2265) |
[docs/outlook/outlook-add-ins-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/outlook/outlook-add-ins-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/outlook-add-ins-overview?branch=pr-en-us-2265) |
[docs/overview/explore-with-script-lab.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/overview/explore-with-script-lab.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/overview/explore-with-script-lab?branch=pr-en-us-2265) |
[docs/overview/learning-path-beginner.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/overview/learning-path-beginner.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/overview/learning-path-beginner?branch=pr-en-us-2265) |
[docs/overview/learning-path-transition.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/overview/learning-path-transition.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/overview/learning-path-transition?branch=pr-en-us-2265) |
[docs/overview/office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/overview/office-add-ins.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/overview/office-add-ins?branch=pr-en-us-2265) |
[docs/overview/patterns-and-practices-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/overview/patterns-and-practices-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/overview/patterns-and-practices-overview?branch=pr-en-us-2265) |
[docs/overview/set-up-your-dev-environment.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/overview/set-up-your-dev-environment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/overview/set-up-your-dev-environment?branch=pr-en-us-2265) |
[docs/powerpoint/powerpoint-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/powerpoint/powerpoint-add-ins.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/powerpoint/powerpoint-add-ins?branch=pr-en-us-2265) |
[docs/project/project-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/project/project-add-ins.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/project/project-add-ins?branch=pr-en-us-2265) |
[docs/resources/resources-links-help.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/resources/resources-links-help.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/resources/resources-links-help?branch=pr-en-us-2265) |
[docs/word/word-add-ins-core-concepts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/word/word-add-ins-core-concepts.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/word/word-add-ins-core-concepts?branch=pr-en-us-2265) |
[docs/word/word-add-ins-programming-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/devprogrammentions/docs/word/word-add-ins-programming-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/word/word-add-ins-programming-overview?branch=pr-en-us-2265) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C10%5C20%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202010201909556811-2265%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=%2FW%2F23dTdrkP7vJjrkmzL191LKa59CQWFY72oAhdlxEk%3D&st=2020-10-20T19%3A05%3A37Z&se=2020-11-20T19%3A10%3A37Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Pull request merged**